great_expectations/agent/agent_server_session.py

```diff
@@ -0,0 +1,27 @@
+import pydantic
+from requests import Session
+
+from great_expectations.core.http import create_session
+
+
+class GxAgentServerSessionConfig(pydantic.BaseSettings):
+    gx_cloud_base_url: str = "https://api.greatexpectations.io"
```

Comment:
```suggestion
    gx_cloud_base_url: str = CLOUD_DEFAULT_BASE_URL
```
https://github.com/great-expectations/great_expectations/blob/70198abacaf7f0713400f64b87c8f51a8f04b137/great_expectations/data_context/cloud_constants.py#L5

Let's use the constant we define for this.

Reply:
great idea, thanks!